swap_meet/item.py

```diff
@@ -1,2 +1,22 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category="", condition=0, age=0):
+        self.category = category
+        self.condition = condition
+        self.age = age
+
+    def __str__(self):
+        return "Hello World!"
+
+    def condition_description(self):
+        if self.condition >= 5:
+            return "This item is brand new!"
+        elif self.condition >= 4:
+            return "This item is slightly worn, but still in good shape."
+        elif self.condition >= 3:
+            return "This item is clearly secondhand, but not too bad."
+        elif self.condition >= 2:
+            return "This item needs cleaning, but might still be OK."
+        elif self.condition >= 1:
+            return "This item is in pretty bad shape."
+        else:
+            return "This item is so old and worn, it's almost unrecognizable."
```

Comment:
Smart ordering of conditionals! We could actually make all the `elif`s just `if`s because we know that the function always returns if it enters a conditional.